lib/plugins/aws/invokeLocal/index.js

```diff
@@ -15,14 +15,30 @@ const download = require('download');
 const mkdirp = require('mkdirp');
 const cachedir = require('cachedir');
 const jszip = require('jszip');
+const ServerlessError = require('../../../classes/Error').ServerlessError;
+const memoize = require('memoizee');
 
 const cachePath = path.join(cachedir('serverless'), 'invokeLocal');
 
+const envVarResolvers = {
+  'AWS::S3::Bucket': require('./resolvers/s3Bucket'),
+  'AWS::DynamoDB::Table': require('./resolvers/dynamoDbTable'),
+  'AWS::SNS::Topic': require('./resolvers/snsTopic'),
+  'AWS::SQS::Queue': require('./resolvers/sqsQueue'),
+  'AWS::Lambda::Function': require('./resolvers/lambdaFunction'),
+  'AWS::SecretsManager::Secret': require('./resolvers/secretsManagerSecret'),
+  'AWS::SSM::Parameter': require('./resolvers/ssmParameter'),
+  'AWS::StepFunctions::StateMachine': require('./resolvers/stepFunctionsStateMachine'),
+};
+
 class AwsInvokeLocal {
   constructor(serverless, options) {
     this.serverless = serverless;
     this.options = options || {};
     this.provider = this.serverless.getProvider('aws');
+    this.loadExports = memoize(this.loadExportsImpl);
+    this.resolveResource = memoize(this.resolveResourceImpl);
```

Comment:
There are two issues with this.  First this would conflict with not calling the memoize version of the function in a comment below.  Second, I get a compile error when I try to create a property at the class level, which is why I fell back to using the constructor.

Reply:
> Second, I get a compile error

What _compile_ error exactly? I do not see it, when I try it on my side

Reply:
I'm probably writing the code wrong, can you put in a code snippet of how you did it?

Reply:
Sure, I've added it here: https://github.com/serverless/serverless/pull/7147#discussion_r368856948

In test example there's a lodash memoizer, but it doesn't make any difference, it'll work same way with `memoizee`